app/src/main/java/org/sopt/sample/SelfIntroduceActivity.kt

```diff
@@ -0,0 +1,28 @@
+package org.sopt.sample
+
+import android.app.Activity
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import org.sopt.sample.databinding.ActivitySelfIntroduceBinding
+import org.sopt.sample.databinding.ActivitySignInBinding
+import org.sopt.sample.databinding.ActivitySignUpBinding
+
+class SelfIntroduceActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySelfIntroduceBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySelfIntroduceBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val id = intent.getStringExtra("id")
+        val mbti = intent.getStringExtra("mbti")
+
+        binding.tvGetName.text= "$id"
+        binding.tvGetMbti.text= "$mbti"
```

Comment:
바로 위에서 언급한 `null`에 대한 처리가 보장된다면, 이렇게 코드를 줄여도 괜찮겠네여

```suggestion

        binding.tvGetName.text= intent.getStringExtra("id")
        binding.tvGetMbti.text= intent.getStringExtra("mbti")
```